docs/blog/posts/2024/srl-vlans2.md

```diff
@@ -0,0 +1,765 @@
+        vlan {
+            encap {
+                single-tagged {
+                    vlan-id 10
+                }
+            }
+        }
+    }
+}
+
+interface ethernet-1/10 {
+    admin-state enable
+    vlan-tagging true
+    subinterface 10 {
+        type bridged
+        admin-state enable
+        vlan {
+            encap {
+                single-tagged {
+                    vlan-id 10
+                }
+            }
+        }
+    }
+}
+
+network-instance bridge-1 {
+    type mac-vrf
+    admin-state enable
+    interface ethernet-1/1.10 {
+    }
+    interface ethernet-1/10.10 {
+    }
+}
+
+```
+
+///
+
+And this is the required config for EOS:
+
+/// details | EOS Configuration
+    type: tip
+
+``` srl
+vlan 10
+!
+interface Ethernet1
+   switchport trunk allowed vlan 10
+   switchport mode trunk
+!
+interface Ethernet10
+   switchport trunk allowed vlan 10
+   switchport mode trunk
+!
+```
+
+///
+
+To apply this configuration we run these two commands:
+
+```bash
+./set-config-srl.sh single-tag
+./set-config-eos.sh single-tag
+```
+
+Let's run the pinger to see how this affects our setup:
+
+```diff
+❯ ./ping-from-cli1.sh all
+- Ping to 10.1.0.2 (no tag) failed.
+- Ping to 10.1.1.2 (single tag VID: 1) failed.
++ Ping to 10.1.10.2 (single tag VID: 10) was successful.
+- Ping to 10.1.11.2 (single tag VID: 11) failed.
+- Ping to 10.1.12.2 (double tag outer VID: 12, inner VID: 13) failed.
+```
+
+As per our [single-tagged VLAN](#single-tagged-vlan) section of the original blog post:
+
+1. Untagged frames are dropped, since they don't have a VLAN
+2. Single tagged frames with `VLAN 1` are dropped, since they don't match the configured `VLAN 10`.
+3. Tagged frames with `VLAN 10` are accepted,  At SRL switch, VLAN tag is popped on ingress and pushed on egress.
+4. Single tagged frames with `VLAN 11` are dropped, since they don't match the configured `VLAN 10`.
+5. Double tagged frames with outer `VLAN 12` are dropped, since the outer tag doesn't match the configured `VLAN 10`.
+
+This is the visual representation of what's happening:
+
+<div class="mxgraph" style="max-width:100%;border:1px solid transparent;margin:0 auto; display:block;" data-mxgraph='{"page":2,"zoom":2,"highlight":"#0000ff","nav":false,"check-visible-state":true,"resize":true,"url":"https://raw.githubusercontent.com/srl-labs/srlinux-eos-vlan-handling-lab/diagrams/srl-eos-vlan.drawio"}'></div>
+
+### Scenario 3: Single-tagged-range VLAN
+
+Whenever we want the subinterface to accept frames with a range of VLAN IDs, we can use single-tagged-range encapsulation type.
+
+This mode doesn't pop any VLAN tags on ingress, nor it adds them on ingress. Subinterface will just filter the frames that have VLAN ID within the configured range and pass them through without any modifications.
+
+In this case, for the SR Linux `ethernet-1/10` interface, we have configured the subinterface with vlan-id `any`. So we can also check that tags won't be modified in any way.
+
```

Comment:
@michelredondo why was this required? We originally had `vlan-tagging false` for inter switch links, is there a point in configuring this to `any`?

Reply:
It's just to show an alternate configuration that provides vlan tagging at interface level and frame transparency.
Maybe we can add something like:
"in the previous post we used `vlan-tagging false' for inter switch links. In this scenario we are using vlan-id `any`, which also provides frame transparency, and it also allows us to use vlan tagging for a specific VLAN ID, at other subinterface in the same port".

Reply:
understood, thanks
added this in [e2600e5](https://github.com/srl-labs/learn-srlinux/pull/146/commits/e2600e50a2d0667cc49c08bca0c9ac8048008a36)